src/pseudopeople/constants/metadata.py

```diff
@@ -99,3 +99,13 @@ class __DateFormats:
 INT_COLUMNS = ["age", "wages", "mailing_address_po_box"]
 
 YEAR_AGGREGATION_VALUE = 3000  # value for all years in a dataset for metadata proportions
+GUARDIAN_DUPLICATION_ADDRESS_COLUMNS = [
+    "street_number",
```

Comment:
nit: use the Columns.name attribute

Reply:
Circular import should usually be resolvable by better encapsulation but I can't think offhand how we might split stuff up more/better than we currently do. Maybe there's a bigger design problem.

Reply:
Consider creating a separate file for "noise-type metadata" including this and `COPY_HOUSEHOLD_MEMBER_COLS`. See if that resolves the circular import.